src/reanimated2/hook/useComposedEventHandler.ts

```diff
@@ -0,0 +1,52 @@
+import { useEvent } from './useEvent';
+import { useHandler } from './useHandler';
+import { WorkletEventHandler } from '../WorkletEventHandler';
+import type { ReanimatedEvent } from './commonTypes';
+import type { WorkletFunction } from '../commonTypes';
+
+/**
+ * Lets you compose multiple event handlers based on [useEvent](https://docs.swmansion.com/react-native-reanimated/docs/advanced/useEvent) hook.
+ *
+ * @param handlers - A list of event handlers created using [useEvent](https://docs.swmansion.com/react-native-reanimated/docs/advanced/useEvent) hook.
+ * @returns An object you need to pass to any "onEvent" prop on an `Animated` component.
+ */
+export function useComposedEventHandler<
+  Context extends Record<string, unknown>
+>(handlers: { workletEventHandler: typeof WorkletEventHandler }[]) {
+  const workletsRecord: Record<string, WorkletFunction> = {};
+  const composedEventNames = new Set<string>();
+  const workletsMap: {
```

Comment:
Is the reason for this being an actual object instead of map the need to copy it to the UI runtime? If not, let's just use a map here.

Reply:
@tjzel Yes, exactly, we need it on the UI runtime